### ISSUE TITLE
Global Facades task: bash not found error

### ISSUE THREAD

**Issue by fgibaux** [laravel-shift/workbench-issues#59]
**Desktop:**
 - OS: Linux
 - OS Version: Fedora 34
 - App Version: 0.7.3

**Describe the bug**
when trying to run the Global Facades task, it does nothing and says `bash: not found`

**To Reproduce**
1. search for facade task
2. Click on `global facades` task
3. Run it

in the log the following appears:
```
Running: laravel/global-facades
0s
sh: bash: not found
sh: bash: not found
sh: bash: not found
sh: bash: not found
```
![image](https://user-images.githubusercontent.com/1101218/153197988-9b87c0f7-e31b-4e22-bbd3-8f9f2549991f.png)

**Comment by jasonmccreary** [comment]
Seems like a bad build. I'll get a rebuild out tomorrow.

**Comment by jasonmccreary** [comment]
@fgibaux we just released 0.7.4. It seems to have resolved the issue for me locally. But please check yourself and close if so. 👍

**Comment by fgibaux** [comment]
yes it did, thank you!
